mtp_cashbook/assets-src/javascripts/modules/address-picker.js

```diff
@@ -0,0 +1,30 @@
+// Address picker module
+'use strict';
+
+exports.AddressPicker = {
+  init: function () {
+    this.bindAddressSelection();
+  },
+
+  bindAddressSelection: function () {
+    var addressSelect = $('#address-select');
+
+    var addressLine1 = $('#id_address_line1');
+    var addressLine2 = $('#id_address_line2');
+    var city = $('#id_city');
+    var postcode = $('#id_postcode');
+
+    function update () {
+      if (addressSelect.val() > 0) {
```

Comment:
the value is a string, so i'd personally use `if (addressSelect.val())` and remove `value="0"`